ethstore/src/lib.rs

```diff
@@ -36,7 +36,6 @@ extern crate ethereum_types;
 extern crate ethkey as _ethkey;
 extern crate parity_wordlist;
 
-#[macro_use]
 extern crate log;
```

Comment:
Are we using `log` crate in `ethstore` at all? If not, maybe remove this altogether?

Reply:
No, not after the change in this PR:  https://github.com/paritytech/parity-ethereum/blob/43c12d676d4b5fad7b4964d5d55b477087f33c86/ethstore/src/accounts_dir/disk.rs#L210

Reply:
@niklasad1 `eprintln!` is from `std`. I see that `macro_use` is completely removed so we aren't using any logging macros from `log` crate?

Reply:
Ah I see, you meant that we should change back to use `log` macros instead for those `eprintln`.

Reply:
I also opt to remove `log` crate since it is not used any more. Unless you want me to restore `warn!` where I've added `eprintln`